src/main/archetype/it.tests/src/main/java/it/tests/CreatePageIT.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright 2015 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package ${package}.it.tests;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.assertion.CQAssert;
+import com.adobe.cq.testing.junit.rules.CQAuthorClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+import com.adobe.cq.testing.junit.rules.Page;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+public class CreatePageIT {
```

Comment:
I would add exhaustive javadocs for this class to help the reader understand every line and it's purpose.

Reply:
Fixed at 49526ac.